app/routes/admin.py

```
@@ -5328,18 +5233,19 @@ def store_management():
         .limit(10)
         .all()
     )
-    pending_redemptions = [
-        SimpleNamespace(
+    pending_redemptions = []
+    for event in pending_redemption_events:
+        seat = db.session.get(Seat, event.seat_id)
+        profile = seat.identity_profile if seat else None
+        grant = _latest_entitlement_grant(event.entitlement_id)
```

Ruff (0.16.1)

[error] Undefined name `_latest_entitlement_grant`

(F821)